round4/Ideation.md

```diff
@@ -0,0 +1,30 @@
+# Chrome Extension Idea: Currency Converter
+## Authors
+Rachel Nguyen, Peter Proenca, Hassan Rizwan, Grant Wells
+
+## Problem Statement
+Travelers, online shoppers, and professionals working across countries often face the challenge of dealing with multiple currencies. Calculating exchange rates can be tedious, error-prone, and time-consuming, especially when dealing with less familiar currencies. Our Chrome Extension simplifies this process by providing instant, accurate, and easy-to-access currency conversion directly in the browser. It eliminates the need to navigate away from the current webpage or use separate apps/tools, streamlining the user's workflow and enhancing their online experience.
+
+## Target Audience
+The target audience for our Chrome Extension includes frequent travelers, expatriates, international students, online shoppers buying from foreign websites, and professionals dealing with cross-border transactions. This audience primarily consists of individuals who regularly require quick and reliable currency conversion to budget, plan, and make informed decisions related to their international activities.
+
+## Description
+Our Chrome Extension offers a quick and easy way to convert currencies right from the browser toolbar. Users can select their base and target currencies, input the amount they wish to convert, and receive real-time conversion rates. The extension also features a customizable watchlist for monitoring favorite currencies and historical rate charts for making informed decisions.
```

Comment:
I think this is a great idea to accomplish for the next 4 weeks. Think about some stretch goals that you could also have to enhance this conversion. Maybe knowing where you are if you are traveling internationally to automatically change it?